image/image.go

```diff
@@ -33,12 +33,13 @@ type genericImage struct {
 	// this field is valid only if cachedManifest is not nil
 	cachedManifestMIMEType string
 	// private cache for Signatures(); nil if not yet known.
-	cachedSignatures [][]byte
+	cachedSignatures      [][]byte
+	supportedDestMIMEType []string
```

Comment:
Actually isn’t this more `requestedManifestMIMETypes`? the `Image` has no concept of a destination, and the MIME types mostly affect what we read and serve.